README.md

```diff
@@ -50,7 +50,7 @@ const zaproxy = new ZapClient(zapOptions);
 Callbacks:
 
 ```js
-zaproxy.spider.scanAsUser(contextId, userId, sutBaseUrl, maxChildren, recurse, subtreeonly, (err, resp) => {
+zaproxy.spider.scanAsUser(argsObject, (err, resp) => {
```

Comment:
Better pass the arguments as an object rather than replace them all.

Reply:
You mean like this `{contextId, userId, setBaseUrl, maxChildren, recurse, subtreeonly}`?

Reply:
Yes.

Reply:
(Tweaking to match the name of the API arguments though.)